application/models.py

```diff
@@ -12,707 +12,1112 @@
+        [
+            ("keywords_en", blocks.CharBlock()),
+        ],
+        null=True,
+        blank=True,
+        verbose_name="keywords EN",
+    )
 
     content_panels = [
         MultiFieldPanel(
             [
-                FieldPanel('heading_section_fi'),
-                FieldPanel('heading_section_sv'),
-                FieldPanel('heading_section_en'),
+                FieldPanel("heading_section_fi"),
+                FieldPanel("heading_section_sv"),
+                FieldPanel("heading_section_en"),
             ],
             heading="Ingressi",
-            help_text='Tämä teksti tulee sivun ylälaitaan ja siihen on tarkoitus kirjoittaa lyhyesti sisällön ydin.',
+            help_text="Tämä teksti tulee sivun ylälaitaan ja siihen on tarkoitus kirjoittaa lyhyesti sisällön ydin.",
         ),
         MultiFieldPanel(
             [
-                FieldPanel('content_section_fi'),
-                FieldPanel('content_section_sv'),
-                FieldPanel('content_section_en'),
+                FieldPanel("content_section_fi"),
+                FieldPanel("content_section_sv"),
+                FieldPanel("content_section_en"),
             ],
             heading="Sisältöäalue",
-            help_text='Tämä sisältö tulee ingressin jälkeen.',
+            help_text="Tämä sisältö tulee ingressin jälkeen.",
         ),
         MultiFieldPanel(
             [
-                StreamFieldPanel('keywords_fi'),
-                StreamFieldPanel('keywords_sv'),
-                StreamFieldPanel('keywords_en'),
+                StreamFieldPanel("keywords_fi"),
+                StreamFieldPanel("keywords_sv"),
+                StreamFieldPanel("keywords_en"),
             ],
             heading="Keywords",
-            help_text='',
+            help_text="",
         ),
     ]
 
-    edit_handler = TabbedInterface([
-        ObjectList(content_panels, heading='Sisältö'),
-        ObjectList(CUSTOM_SETTINGS_PANELS, heading='Asetukset', classname='settings'),
-    ])
+    edit_handler = TabbedInterface(
+        [
+            ObjectList(content_panels, heading="Sisältö"),
+            ObjectList(
+                CUSTOM_SETTINGS_PANELS, heading="Asetukset", classname="settings"
+            ),
+        ]
+    )
 
     def clean(self):
-        self.title = 'Saavutettavuus'
+        self.title = "Saavutettavuus"
         self.slug = str(uuid4())
         super().clean()
 
     def get_context(self, request):
         context = super().get_context(request)
-        context['FRONTEND_BASE_URL'] = settings.FRONTEND_BASE_URL
+        context["FRONTEND_BASE_URL"] = settings.FRONTEND_BASE_URL
         return context
 
     class Meta:
-        verbose_name = 'Accessibility Page'
+        verbose_name = "Accessibility Page"
 
 
 class BannerPages(Page):
-    parent_page_types = ['application.BannerPagesFolder']
+    parent_page_types = ["application.BannerPagesFolder"]
     subpage_typed = []
-
+    preview_modes = []
```

Comment:
Just to make sure because of the format changes pollutes the PR, is this the only real change in this PR :D 
Could be better to split to two commits. One for the real change, one for reformat code.

Reply:
**it is the only change!** The problem is that I had Black formatting on each save. I didn't realise it "changed" so much that it was hard to see the actual changes.